Jenkinsfile

```diff
@@ -30,7 +30,7 @@ pipeline {
             stage('Tests') { steps { sh 'make split-tests -j3'   } }
           }
         }
-        stage('Build') { steps { sh 'make build -j4' } }
+        stage('Build') { steps { sh 'make CC=clang-8 CXX=clang++-8 libcryptopp -j4 && make CC=clang-8 CXX=clang++-8 build -j4' } }
```

Comment:
If these are needed by default, they should be built _into_ the Makefile, not here. The top-level commands like `make build` should "just work" on most systems. And if we can't make it work on most systems then they should "just work" on Ubuntu 18.04.

Reply:
I added another commit that removes these because in my testing, they are not needed on Ubuntu 18.04.